src/coordinator/coordinator.hpp

```diff
@@ -177,11 +177,11 @@ class Coordinator {
 
   /// This adds the provided storage engine to the standby storage engine pool,
   /// which can be used to rebalance storage over time.
-  CoordinatorWriteResponses ApplyWrite(RegisterStorageEngineRequest && /* register_storage_engine_request */);
+  static CoordinatorWriteResponses ApplyWrite(RegisterStorageEngineRequest && /* register_storage_engine_request */);
```

Comment:
I am not sure if these should be static. Shouldn't we just use the clang-tidy suppression comments?

Reply:
yeah, but it's trivial for now to just get it to be quiet with static